apps/simple-trading-app/src/app/components/main/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useRoutes, NavLink } from 'react-router-dom';
+import { Loader, Splash } from '@vegaprotocol/ui-toolkit';
+import {
+  NavigationDrawer,
+  DrawerWrapper,
+  DrawerContainer,
+  DrawerToggle,
+  DRAWER_TOGGLE_VARIANTS,
+} from '../drawer';
+import { routerConfig } from '../../routes/router-config';
+
+export interface RouteChildProps {
+  name: string;
+}
+
+export const AppRouter = () => {
+  const routes = useRoutes(routerConfig);
+
+  const splashLoading = (
+    <Splash>
+      <Loader />
+    </Splash>
+  );
+
+  return (
+    <main className="p-20 overflow-hidden">
+      <React.Suspense fallback={splashLoading}>{routes}</React.Suspense>
```

Comment:
a question (non-blocking): Has Suspense a sense only with lazy import? I can't see it here..